Library/OcConsoleLib/ConsoleGop.c

```diff
@@ -150,51 +150,52 @@ OcProvideConsoleGop (
 
     if (EFI_ERROR (Status)) {
       DEBUG ((DEBUG_INFO, "OCC: No handles with GOP protocol - %r\n", Status));
-      return;
```

Comment:
Do not change the control flow here and increase the indentation level. Return status here instead and update return calls below.